src/popup/pages/AddressBookForm.vue

```diff
@@ -248,6 +252,22 @@ export default defineComponent({
       }
     }
 
+    function shareAddress() {
+      if (
+        isPlaceholder.value
+        || !formModel.value.address
+        || !formModel.value.name
+        || !protocol.value) {
+        return;
+      }
```

Comment:
```suggestion
        || !protocol.value
      ) {
        return;
      }
```

Also I don't think "early return" is necessary here as there is only one function fired. I think early returns should be used only to avoid too many nested ifs.